src/scripts/main.js

```diff
@@ -1,5 +1,43 @@
 'use strict';
 
 document.addEventListener('click', e => {
-  // write code here
+  const wall = document.querySelector('.wall');
+  const spider = document.querySelector('.spider');
+  const wallWidth = wall.clientWidth;
+  const wallHeight = wall.clientHeight;
+  const spiderWidth = spider.clientWidth;
+  const spiderHeight = spider.clientHeight;
+
+  const clickCoordX = Math.ceil(
+    e.clientX - wall.getBoundingClientRect().left - wall.clientLeft
+  );
+  const clickCoordY = Math.ceil(
+    e.clientY - wall.getBoundingClientRect().top - wall.clientTop
+  );
+
```

Comment:
to find the position of mouse click try to use event.offsetX and event.offsetY instead of clientX with getBoundingClientRect()
(these offset props shows coords relative to the target element, 'wall' in our case)

Reply:
and I think Math.ceil is unnecessary here, we need an exact number

Reply:
Про event.offsetX і event.offsetY знаю, в умові було написано про використання clientX/Y. тому робила так.
https://prnt.sc/v2w3m4 
Перероблю